packages/sui-js/README.md

```diff
@@ -5,6 +5,22 @@
 $ npm install @s-ui/js --save
 ```
 
+## pipe
+Consist of a chain of processing functions, where the output of each element is the input of next
+
+```js
+import pipe from @s-ui/js/lib/pipe
+
+execute = async search => {
+  const pageName = await this.pipe(
```

Comment:
Could you remove `this.`from here?. Just a detail to call pipe import

Reply:
ups, sorry, when I created the method, I had the function in the same file and it stayed there by mistake